scheduler/task_queue_persister.go

```diff
@@ -37,6 +37,7 @@ func (self *DBTaskQueuePersister) PersistTaskQueue(distro string,
 			Project:             t.Project,
 			ExpectedDuration:    expectedTaskDuration,
 			Priority:            t.Priority,
+			GroupName:           t.TaskGroup,
```

Comment:
Will this work without adding a bson tag for the group name?

Reply:
There is a a tag. I added the field and its tag in a previous commit. this just saves group information in the queue.